src/grpc/GrpcStatus.js

```diff
@@ -89,43 +89,43 @@ export default class GrpcStatus {
     toString() {
         switch (this) {
             case GrpcStatus.Ok:
-                return "OK";
+                return `OK (${this._code})`;
```

Comment:
Looking the other classes like `Status` for example and to preserve code consistency, `toString()` should return only name of the gRPC error. If it's necessary to build the this format `Status: OK, Code: 0` it could be achieved like this `Status: ${error.status.toString()}, Code: ${error.status.valueOf()}`

Reply:
updated